File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOAddress.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOAddress 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOAddress.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClAddress address)
	   {
            return DAOBD.DAOAddress.SelectOne(address); 
	   }
	   public static int sqlInsert(ClAddress address)
	   {
            return DAOBD.DAOAddress.sqlInsert(address);
       }
	   public static int sqlDelete(ClAddress address)
	   {
            return DAOBD.DAOAddress.sqlDelete(address);
       }
	   public static int sqlUpdate(ClAddress address)
	   {
            return DAOBD.DAOAddress.sqlUpdate(address);
       }
   }
}

File: Sakila_Net/VB_SakilaLib/DAO/DAOAddress.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOAddress
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOAddress
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOAddress.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( address as ClAddress) as Boolean
                        Return DAOBD.DAOAddress.SelectOne(address)     
                    End Function
                    Public Shared  Function SqlInsert( address as ClAddress) as Integer
                        Return DAOBD.DAOAddress.SqlInsert(address)     
                    End Function
                    Public Shared  Function SqlDelete( address as ClAddress) as Integer
                        Return DAOBD.DAOAddress.SqlDelete(address)     
                    End Function
                    Public Shared  Function SqlUpdate( address as ClAddress) as Integer
                        Return DAOBD.DAOAddress.SqlUpdate(address)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
